Rpackage/R/growtree.R

```diff
@@ -0,0 +1,72 @@
+growtree <- structure(function(target.mat, feature.mat, depth=0, maxdepth = Inf,
+                               margin = 0.0, loss="hinge", id = 1L, min_sample = 1,
+                               pred=NULL, side = NULL, weights = NULL) {
+  
+  pid <- NULL
```

Comment:
Can you change pid to something a bit more informative? It's really not clear that it's the node info.

Reply:
done